fava/api/__init__.py

```diff
@@ -451,9 +451,9 @@ def linechart_data(self, account_name):
         } for journal_entry in journal if 'balance' in journal_entry.keys()]
 
     def source_files(self):
-        return list(set(
```

Comment:
A `set` is unordered, using it to remove the duplicate main source file causes the ordering to get lost.